test/systemtest/englishbreakfast/test_sim_verilator.py

```diff
@@ -0,0 +1,263 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import re
+import subprocess
+from pathlib import Path
+
+import pytest
+
+from . import config
+from .. import utils
+
+log = logging.getLogger(__name__)
+
+
+class VerilatorSimEnglishbreakfast:
```

Comment:
just curious: could we derive this class from `VerilatorSimEarlgrey` to reduce duplication?

Reply:
Thanks for your review Michael. I wanted to get this working first before caring about code duplication. But it's a good idea. I've now  reorganized this a bit:
- All config files are moved to either `/test/systemtest/earlgrey` or `/test/systemtest/englishbreakfast`
- There is a `test/systemtest/test_sim_verilator_opentitan.py` that contains the `VerilatorSimOpenTitan` class and function definitions that can be shared. The `test/systemtest/englishbreakfast|earlgrey/test_sim_verilator.py` import this and just define the functions which are specific for the top level.